### PR TITLE
Add clear table selection method

### PR DESCRIPTION
**Pull request by ronen770**
Allow to programmatically clear table rows selected. Such as #214

### CODE REVIEW

File: client/packages/openblocks/src/comps/comps/tableComp/tableComp.tsx [hunk 1]
```diff
@@ -420,6 +420,17 @@ TableTmpComp = withMethodExposing(TableTmpComp, [
       }
     },
   },
+  {
+    method:{
+      name: "clearSelections",
```
Comment (ggbond2077) [inline client/packages/openblocks/src/comps/comps/tableComp/tableComp.tsx]: The method reset the selection to the default value,  so it better rename to "resetSelections"

File: client/packages/openblocks/src/comps/comps/tableComp/tableComp.tsx [hunk 2]
```diff
@@ -420,6 +420,17 @@ TableTmpComp = withMethodExposing(TableTmpComp, [
...
+      description: "",
+      params: [],
+    },
+    execute: (comp) => {
+      comp.children.selection.children.selectedRowKey.dispatchChangeValueAction("0");
```
Comment (ggbond2077) [inline client/packages/openblocks/src/comps/comps/tableComp/tableComp.tsx:430]: Or change the value to an empty string if you want to clear the selection.

File: client/packages/openblocks/src/comps/comps/tableComp/tableComp.tsx (hunk 1 above)
Comment (ronen770) [reply client/packages/openblocks/src/comps/comps/tableComp/tableComp.tsx]: Thanks, changed the method name

**Comment by neon-balcony** [comment]
Hey @ronen770, thanks very much for the PR! Would you please sign the above Contributor License Agreement so that we can merge your PR?
image

**Comment by neon-balcony** [comment]
PR merged, thanks again for the submission! 😄  @ronen770

**Pull request merged**